client/src/client/database_manager_client.py

```diff
@@ -5,11 +5,98 @@
 """
 
 import logging
+from datetime import datetime
+from pathlib import Path
+from typing import Tuple
+
+from playhouse.sqlcipher_ext import SqlCipherDatabase
+
+from client.database_models import (
```

Comment:
Please change to relative import, i.e. "from .database_models"

Reply:
Done.